src/navigation/components/DesktopNavigation/DesktopNavigation.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import _ from 'lodash';
+
+import {block} from '../../../utils';
+import {DesktopNavigationProps} from '../../models';
+import Logo from '../Logo/Logo';
+
+import {LeftItemsWrapper} from './components/LeftItemsWrapper/LeftItemsWrapper';
+import {MobileMenuButton} from './components/MobileMenuButton/MobileMenuButton';
+import {RightItemsWrapper} from './components/RightItemsWrapper/RightItemsWrapper';
+
+import './DesktopNavigation.scss';
+
+const b = block('desktop-navigation');
+
+const DesktopNavigation: React.FC<DesktopNavigationProps> = ({
+    logo,
+    leftItemsWithIconSize,
+    rightItemsWithIconSize,
+    isSidebarOpened,
+    onSidebarOpenedChange,
+    ...props
+}) => (
+    <div className={b('wrapper')}>
+        {logo && (
+            <div className={b('left')}>
+                <Logo {...logo} className={b('logo')} />
+            </div>
+        )}
+        <div className={b('navigation-container')}>
+            <LeftItemsWrapper
```

Comment:
We can left here
`<OverflowScroller className={b(null, className)} onScrollStart={onActiveItemChange}>` 
You don't need `LeftItemsWrapper` and `RightItemsWrapper`